add-cli-auth-to-certificate

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env ruby
+#!/usr/bin/env /opt/puppetlabs/puppet/bin/ruby
```

Comment:
`env(1)` was used to search for a `ruby` command in all paths of `$PATH`.  If we put a full path, it is not needed:
```suggestion
#!/opt/puppetlabs/puppet/bin/ruby
```

Reply:
I've reverted the script back to its original form and updated the README.